cmake/modules/AddSwiftUnittests.cmake

```diff
@@ -10,6 +10,9 @@ function(add_swift_unittest test_dirname)
   # function defined by AddLLVM.cmake.
   add_unittest(SwiftUnitTests ${test_dirname} ${ARGN})
 
+  target_include_directories(${test_dirname} PUBLIC ${LLVM_MAIN_SRC_DIR}/utils/unittest/googletest/include)
+  target_include_directories(${test_dirname} PUBLIC ${LLVM_MAIN_SRC_DIR}/utils/unittest/googlemock/include)
+
```

Comment:
This is incorrect: https://github.com/llvm/llvm-project/blob/main/llvm/cmake/modules/AddLLVM.cmake#L1471

`add_unittest` already links against the library, which should have the public include directories specified, so that would already setup the include paths properly.

Reply:
@compnerd thank you.
now I'm know this should be include header paths too, some reason Xcode project of llvm-project include header but swift Xcode project. i'm continuing to figure out the script.

ps. include_directories is working well too, but want to find more elegant way.

Reply:
I suspect this is only issue for generating Xcode project so now testing some conditional options.

Reply:
@compnerd now, i understand cmake build dependencies from this area. so removing some codes more.